backend/compact-connect/lambdas/license-data/tests/function/test_handlers/test_providers.py

```diff
@@ -277,34 +221,16 @@ def test_query_providers_invalid_sorting(self):
 
         # Should reject the query, with 400
         self.assertEqual(400, resp['statusCode'])
-        self.assertEqual(
```

Comment:
[This discussion](https://github.com/csg-org/CompactConnect/pull/100#discussion_r1688524392) handled here. (Couldn't figure out how to link this change in that discussion, instead of linking that discussion to a comment on this change)